### ISSUE TITLE
Add toolbar button as quick toggle (feature request)

### ISSUE THREAD

**Issue by ya-d** [oblador/hush#111]
TL;DR It would be great having a toolbar button to quickly turn Hush on and off.

I really love Hush is saving me from many annoying cookie banners. On the hand it quite regularly breaks scrolling, display or interactivity of sites I'm visiting.

While it is great to be able to report those issues and have them fixed (thanks a lot!) that’s no instant solution. Currently my workaround is to disable content blocking in website preferences though that is of course overkill and disables ad blocking as well.

Would be great to have a toolbar button to quickly turn Hush on and off. Either generally or for particular sites.
